### PR TITLE
Scores in Label Tooltips

### PR DESCRIPTION
**Pull request by rajadain**
## Overview

Adds score ranges, classification in the score popup.

This could potentially use some design input.

Connects #468 

### Demo

image

Tagging @dboyer for visual review.

### Notes

Need to confirm the Pesticide scores, which we are showing differently than their scale.

## Testing Instructions

* Check out this branch and `beekeepers start`
* Go to [:8000/?beekeepers](http://localhost:8000/?beekeepers) and add some apiaries
* Hover over their scores.
  - Ensure you see the table of expected results in the popup
  - Ensure the scores and their classification is in line with the description in #468

### CODE REVIEW

File: src/icp/apps/beekeepers/js/src/utils.js [hunk 1]
```diff
@@ -313,3 +313,23 @@ export function getOrCreateSurveyRequest({ apiary, id, form }) {
...
+ *
+ * @param {number} score
+ * @param {array} stops
+ */
+export function classifyScore(score, stops) {
+    if (score < stops[1]) {
+        return 'Low';
+    }
+
+    if (score < stops[3]) {
```
Comment (fungjj92) [inline src/icp/apps/beekeepers/js/src/utils.js]: I imagine these should be `<=`

File: src/icp/apps/beekeepers/js/src/components/ScoresLabel.jsx [hunk 2]
```diff
@@ -35,6 +35,31 @@ const ScoresLabel = ({ indicator, scores }) => {
...
+                            <td className="scoreTable__td">{indicatorDetails.scoreStops[1]}</td>
+                            <td className="scoreTable__td">{indicatorDetails.scoreStops[2]}</td>
+                            <td className="scoreTable__td">{indicatorDetails.scoreStops[3]}</td>
+                            <td className="scoreTable__td">{indicatorDetails.scoreStops[4]}</td>
+                        </tr>
+                    </tbody>
+                </table>
+                <div className="scoreVerdict">
+                    {score}
+                    {' = '}
```
Comment (fungjj92) [inline src/icp/apps/beekeepers/js/src/components/ScoresLabel.jsx]: I'll defer to design for styling this. My inclination is the scores should be bold/heavier font-weight but not necessary to change on an inkling.

File: src/icp/apps/beekeepers/js/src/components/ScoresLabel.jsx [hunk 3]
```diff
@@ -35,6 +35,31 @@ const ScoresLabel = ({ indicator, scores }) => {
         >
             <div className="indicator__popup">
                 <div className="name">{indicatorDetails.name}</div>
+                <table className="scoreTable">
+                    <thead>
+                        <tr>
+                            <th className="scoreTable__th">Min</th>
```
Comment (fungjj92) [inline src/icp/apps/beekeepers/js/src/components/ScoresLabel.jsx]: What does `min` mean? Is it the lowest score in the raster ? Minimum score to be viable for bees to live? Same with `max`. Either we need that in the text descriptionsthe client will send, or the min/max should be 0 and 100 respectively.

Comment (rajadain) [reply src/icp/apps/beekeepers/js/src/components/ScoresLabel.jsx]: Hmm, good read, that makes sense. I'll drop the `Min` and the `Max`, and change the UI to indicate that `Low` and lower would be `Low`, and `High` and higher would `high`.

File: src/icp/apps/beekeepers/js/src/components/ScoresLabel.jsx (hunk 2 above)
Comment (rajadain) [reply src/icp/apps/beekeepers/js/src/components/ScoresLabel.jsx]: > but not necessary to change on an inkling

Can you rephrase this? I don't follow.

Comment (fungjj92) [reply src/icp/apps/beekeepers/js/src/components/ScoresLabel.jsx]: Just that I'll defer styling opinions to design

File: src/icp/apps/beekeepers/js/src/components/ScoresLabel.jsx (hunk 3 above)
Comment (rajadain) [reply src/icp/apps/beekeepers/js/src/components/ScoresLabel.jsx]: Is this any better?

image

Comment (fungjj92) [reply src/icp/apps/beekeepers/js/src/components/ScoresLabel.jsx]: Worth presenting to beekeepers in the next test phase ~

**Comment by dboyer** [comment]
This looks good for now. I've sent the screenshot to Penn State to provide the text replacement for the lorum ipsum.
